website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -473,7 +470,7 @@ argument should not be specified.
     `value` parameters that specify header data that will be sent to the origin
     (multiples allowed).
 
-* `origin_access_control_id` (Optional) - The unique identifier of an origin access control for this origin.
+* `origin_access_control_id` (Optional) - The unique identifier of a [CloudFront origin access control][8] for this origin.
```

Comment:
Felt a link to the `aws_cloudfront_origin_access_control` resource doc page was a good idea.